ui/selectors/selectors.js

```diff
@@ -4,9 +4,9 @@ import { SubjectType } from '@metamask/permission-controller';
 import { ApprovalType } from '@metamask/controller-utils';
```

Comment:
This file seems to have been linted or something, is it intended? If not, I would revert it back to its original state. We can lint/do cosmetic changes in another PR probably

Reply:
It was my IDE doing automatic linting! 😮‍💨 I fixed it here [4b2ae4b](https://github.com/MetaMask/metamask-extension/pull/24876/commits/4b2ae4b597e91de9850b4f81ced25eed86ffab25)